generators/app/templates/skill/states.js

```diff
@@ -6,6 +6,14 @@
 */
 
 exports.register = function register(skill) {
+  // This event is triggered after new session has started
+  skill.onSessionStarted((alexaEvent) => {});
```

Comment:
Add reference to the each function http://voxa.readthedocs.io/en/latest/statemachine-skill.html